queries/odin/indents.scm

```diff
@@ -0,0 +1,17 @@
+[
+ (foreign_block)
+ (block)
+ (top_block)
+ (compound_literal)
+ (proc_call)
+ (assignment_statement)
+ (const_declaration)
+ (var_declaration)
+ (switch_statement)
+] @indent
+
+[
+ ")"
+ "}"
+] @indent_end @branch
+
```

Comment:
This is using the old syntax
The equivalents are

- `@indent` -> `@indent.begin`
- `@indent_end` -> `@indent.end`
- `@branch` -> `@indent.branch`